src/bin/identities/CommandSearch.ts

```diff
@@ -54,15 +54,21 @@ class CommandSearch extends CommandPolykey {
             pkClient.grpcClient.identitiesInfoGet(providerMessage, auth),
           meta,
         );
+        let output;
+        if (res.getIdentityId()) {
+          output = [
+            parsers.formatIdentityString(
+              res.getProviderId() as ProviderId,
+              res.getIdentityId() as IdentityId,
+            ),
+          ];
+        } else {
+          output = [`No identities found for provider ${providerId}`];
```

Comment:
You want to do 2 things here:

1. Indicate the sentinel value by simply an empty list. `[]`
2. Additionally write a logger message with `this.logger.info('...');` regarding that no identities were found

Do not write a human readable message in the list item that ism eant to be reserved for the identity string.

Reply:
Done